src/containers/App/App.js

```diff
@@ -32,13 +32,16 @@ class App extends Component {
         }
         <Switch>
           <Route exact path='/' component={NoteArea} />
-          <Route path='/new-note' component={NoteForm} />
+          <Route path='/new-note' render={() => {
+            const note = { title: '', id: Date.now() }
+            return <NoteForm note={note} items={[]} isEdit={false}/>
```

Comment:
Passing in the default props here instead of trying to use an OR in state or trying to sort it out in componentDidMount

Reply:
I agree that having this as a default is more readable than a OR here.

Reply:
👍 I think the same, and it would also be good to switch over to a short id as well